File: Sample Code/ObjcSampleCode/DJISdkDemo/Demo/HandheldController/HandheldControllerActionsTableViewController.m
//
//  HandheldControllerActionsTableViewController.m
//  DJISdkDemo
//
//  Copyright © 2015 DJI. All rights reserved.
//
#import <DJISDK/DJISDK.h>
#import "DemoUtility.h"
#import "HandheldControllerActionsTableViewController.h"
#import "SleepModeViewController.h"

@interface HandheldControllerActionsTableViewController ()

@end

@implementation HandheldControllerActionsTableViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.sectionNames = [NSMutableArray arrayWithArray:@[@"General"]];
    [self.items addObject:@[[DemoSettingItem itemWithName:@"Sleep Mode" andClass:[SleepModeViewController class]]]];
}

-(DJIBaseComponent *)getComponent {
    return [DemoComponentHelper fetchHandheldController]; 
}

@end

File: Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/HandheldController/HandheldControllerActionsTableViewController.swift
//
//  HandheldControllerActionsTableViewController.swift
//  DJISdkDemo
//
//  Copyright © 2015 DJI. All rights reserved.
//
import DJISDK

class HandheldControllerActionsTableViewController: DemoTableViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        self.sectionNames = ["General"]
        self.connectedComponent = ConnectedProductManager.sharedInstance.fetchHandheldController()
        self.showComponentVersionSn = true
        
        self.items.append([DemoSettingItem(name:"Sleep Mode", andClass:SleepModeViewController.self)])
    }
    
}